services/addons/images/count_metric/gen_email.py

```diff
@@ -0,0 +1,103 @@
+import logging
+import os
+import daily_emailer

```

Comment:
We've got a circular reference here with `gen_email` importing `daily_emailer` and `daily_emailer` importing `get_email`. Could we get around this by removing this reference here and instead passing in the values that we want to iterate over via method parameter instead?

Reply:
Removed the circular import. Interesting that I never had an issue with this during testing.

Reply:
Python will happily let you create a circular dependency, but runtime can have odd repercussions